metacat-common-server/src/main/java/com/netflix/metacat/common/server/connectors/model/TableInfo.java

```diff
@@ -39,6 +39,9 @@
 public class TableInfo extends BaseInfo {
     private List<FieldInfo> fields;
     private StorageInfo serde;
+    private String viewOriginalText;
```

Comment:
Add comment saying that the below fields are used when a table is of type 'TABLE_VIEW'.

Reply:
Also I think we should put these three fields in a separate class. Either we have ViewInfo that extends TableInfo or be a member of TableInfo.

Reply:
Also can we get rid of the table type. If viewOriginalText is set, then assume that it is a view.